content/methods/fundamentals/compensation.md

```diff
@@ -1,13 +1,22 @@
 ---
 title: Compensation
+description: Offering usability test or user research participants gifts to encourage participation and to thank them for their time.
```

Comment:
[question] Uh oh, bad news, none of the couple cards that you had done I was riffing off of had description in them, so I deleted this key from all the ones I ported over, though we can get it back easily enough from the "what" text.  Is this actually used for the site metadata?  I haven't looked!

Reply:
tbh I'm not sure the description is really doing anything. I was porting them over in case the purpose revealed itself. We can leave them in there.

Reply:
Or out of there, in the cases where they weren't moved?

Reply:
Indeed, I'll make an issue to decide whether or not we keep them once all porting has been done.